### ISSUE TITLE
Create datum setter node for robot_localization launch

### ISSUE THREAD

**Issue by AbBaSaMo** [Monash-Connected-Autonomous-Vehicle/ESDA#28]
> When the pkg is launched, we need to set the datum via a service. So we'll need a node that reads gps values and calls this service ONCE and in ENU standard where a 0 heading is east
> 
> 
> 
> ![Image](https://github.com/Monash-Connected-Autonomous-Vehicle/ESDA/assets/95030427/15ee2f6c-044a-42f7-b9d2-e309d6697f87)
> 
>  

 _Originally posted by @AbBaSaMo in [#19](https://github.com/Monash-Connected-Autonomous-Vehicle/ESDA/issues/19#issuecomment-1934167259)_

**Comment by AbBaSaMo** [comment]
0 0 0 1 orientation means the robot has a neutral orientation with no rotations, so it seems this is why it is suggested as a starting orientation

**Comment by AbBaSaMo** [comment]
It appears to be working with the set datum as the co-ords of clayton VIC

![Image](https://github.com/Monash-Connected-Autonomous-Vehicle/ESDA/assets/95030427/73b37af0-2aa2-43f5-987d-766e843d5546)

**Comment by AbBaSaMo** [comment]
I misread the docs/tutes on robot_loc. It seems if we set the wait_for_datum parameter as false, the package does not require us to set an initial datum. Everything works even when the datum setter node is not running. If the param is true otherwise, we set the datum as an origin point 

![Image](https://github.com/Monash-Connected-Autonomous-Vehicle/ESDA/assets/95030427/01f0f32c-7718-4193-8233-c1a740af9c61)

Will leave the implementation just in case we decide for whatever reason to set this param as true but closing the issue now.
